File: js/task1/S1_VAR_01.js
var first = "Hello"; // string
let second = 42; // number
const third = true; // boolean

console.table([
  { name: "first", value: first, type: typeof first },
  { name: "second", value: second, type: typeof second },
  { name: "third", value: third, type: typeof third },
]);

File: python/stage1/S1_VAR_01.py
def sample_function():
    pass

values = {
    "int": 10,
    "float": 3.14,
    "str": "hello",
    "bool": True,
    "none": None,
    "list": [1, 2],
    "tuple": (1, 2),
    "dict": {"a": 1},
    "set": {1, 2},
    "function": sample_function
}

print(f"{'name':<10} {'value':<15} {'type(x)':<30} {'type(x).__name__'}")
print("-" * 80)

for name, val in values.items():
    print(f"{name:<10} {str(val):<15} {str(type(val)):<30} {type(val).__name__}")
